controls/roles/manage-service/tasks/ssv-lighthouse/molecule/default/prepare.yml

```diff
@@ -0,0 +1,112 @@
+      when: ansible_distribution == "Ubuntu"
+
+    - name: Install python for Ansible (CentOS 8)
+      raw: yum install -y python38 tar && yum remove -y python36
+      become: true
+      changed_when: false
+      when: ansible_distribution == "CentOS"
+
+    - include_role:
+        name: "setup"
+
+    - name: Create a ssv service directory
+      ansible.builtin.file:
+        path: "/opt/app/services/{{ ssv_service }}"
+        state: directory
+        owner: "2000"
+        group: "2000"
+        mode: '0777'
+      become: yes
+
+    - name: Creating ssv config file
+      copy:
+        dest: "/opt/app/services/{{ ssv_service }}/blox-ssv.yaml"
+        content: |
+          eth2:
+            Network: "prater"
+            BeaconNodeAddr: "http://stereum-{{ beacon_service }}:5052"
+          eth1:
+            ETH1Addr: "MyEth1Node"
+            RegistryContractAddr: "0x687fb596F3892904F879118e2113e1EEe8746C2E"
+          OperatorPrivateKey: ""
+          global:
+            LogLevel: "debug"
+          MetricsAPIPort: 15000
+        owner: "2000"
+        group: "2000"
+        mode: 0777
+      become: yes
+
+    # lh beacon service
+    - block:
+      - set_fact:
+          stereum: "{{ stereum_static | combine(stereum_args, recursive=True) }}"
+      - name: "Include manage-service"
+        include_role:
+          name: "manage-service"
+      vars:
+        stereum_args:
+          manage_service:
+            save: true
+            state: started
+            configuration:
+              id: "{{ beacon_service }}"
+              image: stereum/lighthouse:v2.0.1-48
+              env:
+                DEBUG_LEVEL: debug
+                NETWORK: prater
+                ETH1_NODES: MyEth1Node
+                DATADIR: /opt/app/beacon
+                SLASHERDIR: /opt/app/slasher
+                SLASHER_DB_SIZE: "16"
+              ports:
+                - 0.0.0.0:9000:9000/tcp
+                - 0.0.0.0:9000:9000/udp
+                - 127.0.0.1:5052:5052/tcp
+              command: []
+              entrypoint: ["/opt/app/start/beacon.sh"]
+              user: "2000"
+              volumes:
+                - "/opt/app/services/{{ beacon_service }}/beacon:/opt/app/beacon"
+                - "/opt/app/services/{{ beacon_service }}/slasher:/opt/app/slasher"
+
+    # ssv service
+    - block:
+      - set_fact:
+          stereum: "{{ stereum_static | combine(stereum_args, recursive=True) }}"
+      - name: "Include manage-service"
+        include_role:
+          name: "manage-service"
+      vars:
+        stereum_args:
+          manage_service:
+            save: true
+            state: started
+            configuration:
+              id: "{{ ssv_service }}"
+              image: bloxstaking/ssv-node:latest-ubuntu
+              ports:
+                - 0.0.0.0:12000:12000/udp
+                - 0.0.0.0:13000:13000/tcp
+              command: make BUILD_PATH=/go/bin/ssvnode start-node && docker logs ssv_node
```

Comment:
The goal is to bring the creation of the config file into the command, so we don't need an extra task for the config file creation.

If this is not possible in any way then we need to discuss how to handle config files in the `manage-service` role properly.

Reply:
solved using env & echo